webui/react/src/components/DeterminedAuth.tsx

```diff
@@ -87,6 +87,7 @@ const DeterminedAuth: React.FC<Props> = ({ canceler }: Props) => {
 
   const loginForm = (
     <Form
+      data-test="authForm"
```

Comment:
spelling: `data-testid` I think you mean?

Reply:
actually, i like top level components to have `data-test`. when the form gets implemented in `SignIn.tsx`, sign in could give it a `data-testid`, but it wouldn't need to unless it was instantiating more than one of this component. I'll write this up in a guide somewhere.

Reply:
Why do you like top level components to be slightly different? To me it reads like an extra style thing for maintainers to remember and no implied meaning. 
If you do want the top level to be different, can it be explicit? Something like `data-top-level-testid` or `data-formid`? It'd be good if the name was different enough to indicate that this difference is intentional, and ideally it could also describe why it was different.

Reply:
maybe this?
```
data-test-component='my-component'
```

Reply:
Seems diffferent enough form data-testid to avoid confusion. works for me!